test/pins-test.js

```diff
@@ -0,0 +1,79 @@
+/* eslint max-nested-callbacks: ["error", 8] */
+/* eslint-env mocha */
+'use strict'
+
+const { Buffer } = require('buffer')
+const { expect } = require('./utils/chai')
+const range = require('just-range')
+const Key = require('interface-datastore').Key
+
+module.exports = (repo) => {
+  describe('pins', () => {
+    const dataList = range(100).map((i) => Buffer.from(`hello-${i}-${Math.random()}`))
+    const data = Buffer.from('hello world')
+    const b = new Key('hello')
+
+    it('exists', () => {
+      expect(repo).to.have.property('pins')
+    })
+
+    it('implements interface-datastore', () => {
```

Comment:
This test is redundant as `datastore-level` already runs the interface tests: https://github.com/ipfs/js-datastore-level/blob/master/test/index.spec.js#L44